lldb/source/Target/StackFrame.cpp

```diff
@@ -1230,6 +1230,57 @@ bool StackFrame::IsHidden() {
   return false;
 }
 
+const char *StackFrame::GetFunctionName() {
+  const char *name = nullptr;
+  SymbolContext sc = GetSymbolContext(
+      eSymbolContextFunction | eSymbolContextBlock | eSymbolContextSymbol);
+  if (sc.block) {
+    Block *inlined_block = sc.block->GetContainingInlinedBlock();
+    if (inlined_block) {
+      const InlineFunctionInfo *inlined_info =
```

Comment:
You didn't do this but since you're touching this...  GetInlinedFunctionInfo is not guaranteed to return a non-null pointer.  Other places that access the underlying m_inlineInfoSP always check it against nullptr before accessing it.  Probably a good idea to do that here as well...